cmd/antrea-agent/util.go

```diff
@@ -79,3 +84,25 @@ func parsePortRange(portRangeStr string) (start, end int, err error) {
 
 	return start, end, nil
 }
+
+func getServiceListOptionsModifier(serviceProxyName string) (func(*metav1.ListOptions), error) {
```

Comment:
Please add unit test for new added/changed codes.

Reply:
Will try do add.

Reply:
Done.